frontend/src/app/chart/chart.component.ts

```diff
@@ -46,7 +47,7 @@ export class ChartComponent implements OnInit, OnDestroy {
 
   // Component binding
   @Output() message = new EventEmitter<string>();
-  filename = 'DMM_result_multiple_channel.csv';
+  filename: string;
```

Comment:
Please add ! mark for all variables that you only declare type but not initialize a value. eg: filename!: string